src/services/lesson/hooks/index.js

```diff
@@ -76,7 +76,7 @@ exports.before = () => ({
 	patch: [
 		checkIfCourseGroupLesson.bind(this, 'COURSEGROUP_EDIT', 'TOPIC_EDIT', false),
 		permitGroupOperation,
-		checkCorrectCourseOrTeamId,
+		ifNotLocal(checkCorrectCourseOrTeamId),
```

Comment:
this should use `iff(provider('external')`

Reply:
Was introduced by merge with develop. too risky to change.

Reply:
if it was introduced by the develop branch, then it wouldn't pop up as a change when merging into develop